pybamm/models/submodels/interface/kinetics/base_first_order_kinetics.py

```diff
@@ -26,7 +27,12 @@ def get_coupled_variables(self, variables):
         # Unpack
         c_e_0 = variables["Leading-order average electrolyte concentration"]
         c_e = variables[self.domain + " electrolyte concentration"]
-        c_e_1 = (c_e - c_e_0) / self.param.C_e
+        c_e_1 = (
+            c_e
+            - pybamm.Broadcast(
+                c_e_0, self.domain_for_broadcast, broadcast_type="primary"
```

Comment:
is `variables["Leading-order negative electrolyte concentration"]` equivalent to this? It is for lithium-ion.